internal/util/logger/logger.go

```diff
@@ -20,28 +20,36 @@ const (
 	loggerDefaultName    = "numaplane"
 	messageFieldName     = "msg"
 	timestampFieldName   = "ts"
+	defaultLevel         = InfoLevel
 )
 
+// infoLevelShift defines the value to shift by from the InfoLevel
+const infoLevelShift = 3
+
 /*
 Log Level Mapping:
 | NumaLogger semantic | logr verbosity  | zerolog        |
 | ------------------- | --------------- | -------------- |
-|    fatal            |    0            |    4           |
+|    fatal            |    1            |    4           |
 |    error            |    error (NA)   |    3           |
-|    warn             |    1            |    2           |
-|    info             |    2            |    1           |
-|    debug            |    3            |    0           |
-|    verbose          |    4            |    -2 (custom) |
+|    warn             |    2            |    2           |
+|    info             |    3            |    1           |
+|    debug            |    4            |    0           |
+|    verbose          |    5            |    -2 (custom) |
 */
-
 const (
-	FatalLevel   = 0
-	WarnLevel    = 1
-	InfoLevel    = 2
```

Comment:
If specify `logLevel: 4` in configmap, I should expect Debug level logging, right?

Reply:
Yes, that is the expected behavior.

Reply:
Thanks! I verified that works. However, I also tried to see if log level can be dynamically configured. That is working except one scenario, if the logLevel field is removed from the configmap, we are still setting the [value](https://github.com/numaproj-labs/numaplane/blob/main/internal/sync/syncer.go#L223) instead of using default one. I think this is minor can be address as a follow up. do you mind create an issue to track it? Thanks a lot!